Mathlib/ModelTheory/FinitelyGenerated.lean

```diff
@@ -235,7 +237,7 @@ theorem FG.countable_embedding (N : Type*) [L.Structure N] [Countable N] (_ : FG
     Countable (M ↪[L] N) :=
   Function.Embedding.countable ⟨Embedding.toHom, Embedding.toHom_injective⟩
 
-instance Fg.instCountable_embedding (N : Type*) [L.Structure N]
+instance FG.instCountable_embedding (N : Type*) [L.Structure N]
     [Countable N] [h : FG L M] : Countable (M ↪[L] N) :=
```

Comment:
```suggestion
instance FG.instCountable_embedding (N : Type*) [L.Structure N] [Countable N] [h : FG L M] :
    Countable (M ↪[L] N) :=
```

Reply:
I don't really agree here. What's your rationale?

Reply:
:shrug: 

Fitting the assumptions on one line, the conclusion on the next

Reply:
Oh sorry, I read the diff the wrong around 🤦 I agree with you now